src/app/helpers/isAmpPath.js

```diff
@@ -0,0 +1,5 @@
+const AMP_PATH_REGEX = /^\/news\/articles\/amp\/:id$/;
```

Comment:
Does this need to be hardcoded with `news` or would `/\/articles\/amp\/:id$/` not be acceptable? This way we won't need to remove news or case match the service name in the future.